### DISCUSSION TITLE
alist进行同步如何保留原文件“修改时间”

### DISCUSSION THREAD

**Discussion by ready2race** [AlistGo/alist#3961] [Q&A]
### Alist Version / Alist 版本
v3.14.0

###  Driver used / 使用的存储驱动
本地存储

### Describe the bug / 问题描述

使用raidrive挂载alist到本地后，不管是直接复制文件还是通过good2sync同步，文件的修改时间总会被修改为当前同步的时间，是否有其他方法可以保留原文件的属性？感谢！

![屏幕截图 2023-03-26 163933](https://user-images.githubusercontent.com/30542232/227764887-968623c3-f05a-4465-a74f-19f7070b1ca0.png)

**Comment by DentistCode** [comment]
我也遇见这个问题了，这个可能是alist作为webdav没带这个功能，建议提issue(我还没看)。

我是在使用rclone同步两个盘的时候发现的问题。尝试了rclone、goodsync，alist都为webdav形式。

---

参考一段rclone的doc

普通WebDAV不支持修改时间。但是Fastmail Files，Owncloud或Nextcloud，rclone将支持修改时间。
同样，WebDAV不支持哈希，但是当与Fastmail Files一起使用时，Owncloud或Nextcloud rclone将支持SHA1和MD5哈希。
根据Owncloud或Nextcloud的确切版本，哈希可能出现在所有对象上，或者仅出现在与它们一起上传哈希的对象上。


<details><summary>原文</summary>
<p>


Modification times and hashes
Plain WebDAV does not support modified times. However when used with Fastmail Files, Owncloud or Nextcloud rclone will support modified times.
Likewise plain WebDAV does not support hashes, however when used with Fastmail Files, Owncloud or Nextcloud rclone will support SHA1 and MD5 hashes. Depending on the exact version of Owncloud or Nextcloud hashes may appear on all objects, or only on objects which had a hash uploaded with them.

来自：https://rclone.org/webdav/

</p>
</details>
